docs/self-host.docker

```diff
@@ -0,0 +1,16 @@
+FROM vladikoff/fxa-slim-image:latest
+
+RUN adduser --disabled-password --gecos '' fxa && adduser fxa sudo && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+COPY . /home/fxa/fxa-profile-server
+WORKDIR /home/fxa/fxa-profile-server
+RUN chown -R fxa .
+USER fxa
+
+RUN npm install --production \
+  && npm cache clear
+
+CMD npm start
+
+# Expose ports
+EXPOSE 1111
```

Comment:
Thanks! Re 1, there are no release tags for https://hub.docker.com/r/vladikoff/fxa-slim-image/tags/ other than `latest`, and agree it's not something to worry about probably.

Re 2, I'm happy to add a MAINTAINER field, but if someone wants to contact the maintainer to propose an improvement, I would not be the best person to contact (since I'm not a committer/reviewer); adding this Dockerfile to the repo this is just a drive-by contribution. :) Maybe set MAINTAINER to 'the FxA team'?